.changeset/little-geckos-end.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+---
+
+Add new endpoints to Kubernetes backend plugin
+
+BREAKING: Kubernetes backend plugin now depends on CatalogApi
```

Comment:
For things like this one, it's good to make a `patch` changeset to `create-app` and adding a small diff explanation for how to update your installation accordingly (essentially adding the three lines that you have below). And having the same diff in this file as well doesn't hurt.

Reply:
No that's fine, disregard freben's comment 😅 (I'm assuming it's a backend vs create-app/.../backend mixup)

Do switch this out to `**BREAKING**:` for high visibility thought :grin: